WordPress/Classes/Models/Post.swift

```diff
@@ -204,6 +240,15 @@ class Post: AbstractPost {
 
                 return true
             }
+
+            if publicizeMessage ?? "" != originalPost.publicizeMessage ?? "" {
+                return true
+            }
+
+            if (!NSDictionary(dictionary: disabledPublicizeConnections ?? [:])
+                             .isEqual(to: originalPost.disabledPublicizeConnections ?? [:])) {
+                return true
```

Comment:
Wouldn't that prevent detecting changes when  `disabledPublicizeConnections = nil` ?